From f8aef8ae82831ebfda9520d6e7e82a2cad99383d Mon Sep 17 00:00:00 2001
From: Vincent Untz <vuntz@gnome.org>
Date: Fri, 7 Oct 2011 00:04:17 +0200
Subject: [PATCH] Update categories in brasero.desktop

I don't think Utility (nor Core) should be there, and it exposes a
gnome-shell bug where brasero appears twice because it's in two
categories.

https://bugzilla.gnome.org/show_bug.cgi?id=661134
---
 data/brasero.desktop.in.in |    2 +-
 1 files changed, 1 insertions(+), 1 deletions(-)

diff --git a/data/brasero.desktop.in.in b/data/brasero.desktop.in.in
index ac83297..ce560c2 100644
--- a/data/brasero.desktop.in.in
+++ b/data/brasero.desktop.in.in
@@ -2,7 +2,7 @@
 _Name=Brasero
 _GenericName=Disc Burner and Copier
 _Comment=Create and copy CDs and DVDs
-Categories=GNOME;AudioVideo;DiscBurning;Core;Utility;
+Categories=GTK;GNOME;AudioVideo;DiscBurning;
 MimeType=application/x-cd-image;application/x-cdrdao-toc;application/x-cue;application/x-toc;audio/x-scpls;audio/x-ms-asx;audio/x-mp3-playlist;audio/x-mpegurl;application/x-brasero;x-content/audio-cdda;x-content/video-dvd;x-content/video-vcd;x-content/video-svcd;x-content/image-picturecd;
 Exec=brasero %U
 Icon=brasero
-- 
1.7.7.2

